modules/registry-third-party-registries.adoc

```diff
@@ -5,12 +5,32 @@
 [id="registry-third-party-registries_{context}"]
```

Comment:
As per Jupiter guidelines, we need to add:_content-type: PROCEDURE @kalexand-rh ptal

Reply:
acked